extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js

```diff
@@ -11,7 +11,7 @@ RdfSkeletonListDialog.prototype._createDialog = function() {
     frame.width("800px");
     
     var header = $('<div></div>').addClass("dialog-header").text("RDF Skeleton List").appendTo(frame);
-    var body = $('<div></div>').css('height', '150px').addClass("dialog-body").appendTo(frame);
+    var body = $('<div></div>').css('height', '250px').addClass("dialog-body").appendTo(frame);
```

Comment:
Let's have a discussion at some point where to put the CSS, in code or in a `.css` file.